### PR TITLE
[Image] Document the SVG behavior

### LINKED ISSUE #199: [Image] Document the SVG behavior
We should document the SVG support that was recently added to the image component in https://github.com/Adobe-Marketing-Cloud/aem-core-wcm-components/pull/171, to describe its behavior in relation to adaptive image widths.

### PR DESCRIPTION
**Pull request by richardhand**
| Q                        | A 
| ------------------------ | ---
| Fixed Issues?            | `Fixes #199` 
| Patch: Bug Fix?          |
| Minor: New Feature?      |
| Major: Breaking Change?  |
| Tests Added + Pass?      | N/A
| Documentation Provided   | Yes
| Any Dependency Changes?  |
| License                  | Apache License, Version 2.0

Add documentation in component README.md for SVG support.

### CODE REVIEW

File: content/src/content/jcr_root/apps/core/wcm/components/image/v1/image/README.md [hunk 1]
```diff
@@ -106,6 +107,11 @@ CSS styling. It should be added to a relevant site client library using the `emb
 It also provides a `core.wcm.components.image.v1.editor` editor client library category that includes
 JavaScript handling for dialog interaction. It is already included by its edit dialog.
 
+## SVG
+SVG MIME-types are supported, but have some specific handling. In contrast to lazy loading, which is still possible for SVG image types,
+alternative image sizes defined at the component policy dialog are ignored. In addition, SVG image types have a more limited set of
+editing options available in the AEM inline image editor.
```
Comment (kwin) [inline content/src/content/jcr_root/apps/core/wcm/components/image/v1/image/README.md]: Can you already mention here which minimum AEM Version/Feature Pack is required for the limited editing options to be correctly applied?

Comment (richardhand) [reply content/src/content/jcr_root/apps/core/wcm/components/image/v1/image/README.md]: @kwin - we have plans to ensure the required product changes for both SVG and Image Map support are available on the currently supported AEM versions for the Core Components (6.3, 6.4). The exact feature or service pack is not yet known, but we will be updating our versioning documentation with a subsequent release of the core components.

File: content/src/content/jcr_root/apps/core/wcm/components/image/v1/image/README.md [hunk 2]
```diff
@@ -106,6 +107,10 @@ CSS styling. It should be added to a relevant site client library using the `emb
 It also provides a `core.wcm.components.image.v1.editor` editor client library category that includes
 JavaScript handling for dialog interaction. It is already included by its edit dialog.
 
+## SVG
+SVG MIME-types are supported, but have some specific handling. Alternative image sizes defined in the component policy dialog are ignored for SVG images. In addition, SVG image types have a more limited set of
+editing options available in the AEM inline image editor.
```
Comment (jckautzmann) [inline content/src/content/jcr_root/apps/core/wcm/components/image/v1/image/README.md]: I would be more specific and state that smart sizes are not supported and that Image#getWidths returns an empty array.
We should also add that lazy loading is supported.

Comment (richardhand) [reply content/src/content/jcr_root/apps/core/wcm/components/image/v1/image/README.md]: Thanks @jckautzmann - updated in https://github.com/Adobe-Marketing-Cloud/aem-core-wcm-components/pull/200/commits/6d18fcfe113db5d00a26670dcde8b684613e1a63

**Pull request merged**